api/src/main/java/javax/mvc/binding/BindingResult.java

```diff
@@ -99,31 +99,31 @@
     BindingError getBindingError(String param);
 
     /**
-     * Returns an immutable set of all constraint violations detected.
+     * Returns an immutable set of all validation errors detected.
      *
-     * @return All constraint violations detected
+     * @return All validation errors detected
      */
-    Set<ConstraintViolation<?>> getAllViolations();
+    Set<ValidationError> getAllValidationErrors();
```

Comment:
Hey @Turini. Awesome, thanks for the review.

I basically agree with you, that we should remove the "all" suffix. So +1 for that. But I used this name in the pull request to be consistent with `getAllBindingErrors()`. That's why I also included the "all" suffix in `getAllValidationErrors()`.

IMO we still need to discuss some other method and parameter names. So my suggestion would be to do this in a second step. So first we do these structural changes contained in this pull request in a consistent way (keep "all" as it is also used for binding errors) and after that we should do some alignment and cleanup of method and parameter names. This second step will hopefully be more easy to get feedback about. I think we should try to do small pull requests changing only certain aspects to be able to do reviews more easily. 

Thoughts?